CONTRIBUTING.md

```diff
@@ -33,5 +58,15 @@
 - ➖ `:heavy_minus_sign:` : Deletion of some code
 - ♻️ `:recycle:` : Refactor code
 
-# Extra
-- 🚀 `:rocket:` : Final commit
\ No newline at end of file
+### Extra
+- 🚀 `:rocket:` : Final commit
+
+## Releases
+
+Here we will explain how the files located in the [release page](https://github.com/Tugduoff/RType/releases) of Github are  produced
+
+#### Windows
+On windows we simply launch the build script (`windows_build_rtype.bat`), and then zip the produced bin folder.
```

Comment:
Idk if I should mention it but there is a space between `folder` and `.`

Reply:
Not really